### PR TITLE
feat(dedicated): add guides for 3az cluster

### PR DESCRIPTION
**Pull request by mhelhali-soufien**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/multi-az-feature`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13600
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
  add guides link to 3AZ cluster 
## Related
 #MANAGER-11699
## Translation 
 #CMT-9635

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 export const STATUS = { DONE: 'DONE' };
+export const GUIDES_LINK =
+  'https://github.com/ovh/docs/blob/3az_JTA/pages/bare_metal_cloud/dedicated_servers/3az/guide.en-gb.md';
```
[the remark this answers is not in the document]
Comment (qpavy) [reply packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js:3]: @anooparveti We don't have guides for other languages on https://github.com/ovh/docs/blob/3az_JTA/pages/bare_metal_cloud/dedicated_servers/3az

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 export const STATUS = { DONE: 'DONE' };
+export const GUIDES_LINK =
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js:2]:
```suggestion
export const 3AZ_GUIDE_LINK =
```

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.routing.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { STATUS } from './cluster.constant';
+import { STATUS, GUIDES_LINK } from './cluster.constant';
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.routing.js:1]:
```suggestion
import { STATUS, 3AZ_GUIDE_LINK } from './cluster.constant';
```

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.routing.js [hunk 4]
```diff
@@ -25,6 +25,7 @@ export default /* @ngInject */ ($stateProvider) => {
       trackingPrefix: () => 'dedicated::dedicated::cluster',
       currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
         $state.href($state.current.name, $transition$.params()),
+      guidesLink: () => GUIDES_LINK,
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.routing.js:28]:
```suggestion
      3azGuideLink: () => 3AZ_GUIDE_LINK,
```

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -1,4 +1,6 @@
 {
   "cluster_general_information": "Informations générales",
-  "cluster_nodes": "Noeuds"
+  "cluster_nodes": "Noeuds",
+  "dedicated_cluster_guides_title": "Guides",
+  "dedicated_cluster_guides_link": "3AZ Guides"
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/translations/Messages_fr_FR.json:5]:
```suggestion
  "dedicated_cluster_3az_guide_link": "3AZ Guides"
```

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.html [hunk 6]
```diff
@@ -1,6 +1,19 @@
 <header class="oui-header" role="banner">
-    <div class="d-flex">
+    <div class="d-flex justify-content-between">
         <h1 class="oui-header__title" data-ng-bind="$ctrl.displayName"></h1>
+        <oui-guide-menu
+            class="mt-2"
+            data-text="{{:: 'dedicated_cluster_guides_title' | translate }}"
+        >
+            <oui-guide-menu-item
+                data-ng-href="{{:: $ctrl.guidesLink }}"
```
Comment (qpavy) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.html:9]:
```suggestion
                data-ng-href="{{:: $ctrl.3azGuideLink }}"
```

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js (hunk 2 above)
Comment (mhelhali-soufien) [reply packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/cluster/cluster.constant.js:2]: 3AZ_GUIDE_LINK is not valide name for variable bc we can't start naming with number so I change to _3AZ_GUIDE_LINK

**Review by qpavy — APPROVED** [review]

**Comment by qpavy** [comment]
Already merged here https://github.com/ovh/manager/pull/11174

**Pull request closed without merging**